partner_contact_sale_info_propagation/models/res_partner.py

```diff
@@ -18,67 +15,43 @@ def _check_propagation_allowed(self):
         )
 
     def write(self, vals):
-        """Propagate Salesperson and Sales Channel change in the partner to the
-        child contacts."""
+        """
```

Comment:
The correct docstring format according PEP257 is the previous one: https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
Being strict, the closing `"""` should be put in a new line (not the opening one, which is where I commented), but for this time, it's OK to leave it this way.